exercises/chapter6/src/Main.purs

```diff
@@ -3,8 +3,15 @@ module Main where
 import Prelude
 
 import Effect (Effect)
-import Effect.Console (log)
+import Effect.Class.Console (logShow)
+import Data.Hashable (hash, hashEqual)
 
 main :: Effect Unit
 main = do
-  log "🍝"
+  logShow $ hash 123
+  logShow (hash true)
+  logShow (hash [1, 2, 3])
+  logShow (hash "testing")
+  logShow (hash 'a')
+  logShow ("foo" `hashEqual` "foo")
+  logShow ("foo" `hashEqual` "bar")
```

Comment:
These used to live in `test/Main.purs`, but relocating them here, since the original location seems best for exercise tests.